src/TimeService.Business/Commands/Import/ImportStatCommand.cs

```diff
@@ -148,24 +151,36 @@ private List<Guid> FindDepartmentUsers(Guid departmentId, DateTime filterbyEntry
       return null;
     }
 
-    private List<UserInfo> GetUsers(List<Guid> userIds, List<string> errors)
+    private async Task<List<UserData>> GetUsersData(List<Guid> userIds, List<string> errors)
```

Comment:
Maybe it will be better if you check userIds != null

Reply:
fixed